blivet/tasks/fsresize.py

```diff
@@ -38,13 +34,15 @@
 import logging
 log = logging.getLogger("blivet")
 
+import gi
+gi.require_version("BlockDev", "3.0")
+from gi.repository import BlockDev
+
 
 @add_metaclass(abc.ABCMeta)
```

Comment:
Is the decorator still needed here? More of the edited functions have the abc decorator left in them even though abc is no longer used in them.